src/main/java/rcms/utilities/daqexpert/reasoning/logic/failures/HltOutputBandwidthTooHigh.java

```diff
@@ -0,0 +1,69 @@
+package rcms.utilities.daqexpert.reasoning.logic.failures;
+
+import org.apache.log4j.Logger;
+import rcms.utilities.daqaggregator.data.DAQ;
+import rcms.utilities.daqexpert.ExpertException;
+import rcms.utilities.daqexpert.ExpertExceptionCode;
+import rcms.utilities.daqexpert.Setting;
+import rcms.utilities.daqexpert.reasoning.base.action.SimpleAction;
+import rcms.utilities.daqexpert.reasoning.logic.basic.Parameterizable;
+
+import java.util.Map;
+import java.util.Properties;
+
+public class HltOutputBandwidthTooHigh extends KnownFailure implements Parameterizable {
+
+    /**
+     * upper end of range for expected  rate
+     */
+    private double max;
+
+    private static final Logger logger = Logger.getLogger(HltOutputBandwidthTooHigh.class);
+
+    public HltOutputBandwidthTooHigh() {
+        this.name = "Too high HLT output bandwidth";
+        this.max = 0;
+
+        this.action = new SimpleAction("Are we running with the correct pre-scale column?",
+                "Talk to the trigger shifter and shift leader",
```

Comment:
Talk to the trigger shifter and shift leader. Have them check the pre-scale column. You may need to call the HLT DOC.